apps/nextjs/src/middlewares/csp.ts

```diff
@@ -1,21 +1,14 @@
 import { NextRequest, NextResponse } from "next/server";
+import { v4 as uuidv4 } from "uuid";
```

Comment:
There's an incompatibility somewhere with Turbopack not having access to "crypto". So instead it uses a UUID.